Project-iOS/XLProjectName/XLProjectName/Helpers/Extensions/UIImageView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UIImageView.swift
+//  XLProjectName
+//
+//  Created by XLAuthorName ( XLAuthorWebsite )
+//  Copyright © 2016 'XLOrganizationName'. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+import AlamofireImage
+
+
+extension UIImageView {
+    public func setImageWithURL(url: String, filter: ImageFilter? = nil, placeholder: UIImage? = nil, completion: (Response<UIImage, NSError> -> Void)? = nil) {
```

Comment:
I think we do use AlamofireImage in most of our projects and this is a way to handle image loading errors centralized. It is also useful to have the same animation (CrossDissolve in this case) for every image. And it should not go in XLSwiftKit as we will want to change this in every project.